pages/login.js

```diff
@@ -0,0 +1,12 @@
+export default function Login(){
+  return (
+    <div>
+      <h2>Login</h2>
+      <input type="text" placeholder="Enter email here"/>
+      <input type="password" placeholder="******"/>
+      <button>Login</button>
+      <p>Don't have an account?</p>
+      <a href="#">Sign Up</a>
+    </div>
+  )
+}
```

Comment:
Nitpick - could you add a newline at the end of this file?

[why end files with a newline?](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)